config/vufind/config.ini

```
@@ -2643,3 +2643,25 @@ description = "The REST API provides access to search functions and records cont
 ; By default, VuFind sorts text in a locale-agnostic way; if this setting is
 ; turned on, the current user-selected locale will impact sort order.
 ;use_locale_sorting = true
+
+; Data retrieved from records is escaped by default, on the assumption that it does not contain HTML.
+; This section can be used to allow limited HTML in specific contexts; set values to true to allow HTML,
+; and leave false to escape content.
```

I agree that trying to comprehensively list every context will be hard to maintain... but I think it would help to explain exactly what the contexts represent and where they are used. What about something like this?

```suggestion
; and leave false to escape content. The keys represent data context arguments sent to the
; escapeOrCleanHtml view helper. Some examples are shown below, but the list may not be
; comprehensive; you can also define your own custom contexts for use in local templates.
```

---

@demiankatz Good suggestion, committed!